modules/boss/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+## 4.2.0 - 2021-07-09
+
+### Added
+- Added `property().getLanguage` method to Universe
```

Comment:
```suggestion
- Added `property().getLanguage` function to Universe.
```
Changelogs should either have a JIRA reference or a fullstop at the end. Also method isn't used very often in the JS world, so function is probably more accurate.

Reply:
```suggestion
- Added `property().getLanguage` function to Universe #TINY-7570
```